cmake/templates/ngraphConfig.cmake.in

```diff
@@ -84,5 +84,6 @@ if(ngraph_onnx_importer_FOUND)
 endif()
 
 set(ngraph_paddlepaddle_frontend_FOUND @NGRAPH_PDPD_FRONTEND_ENABLE@)
+set(ngraph_ir_frontend_FOUND @NGRAPH_IR_FRONTEND_ENABLE@)
```

Comment:
I think we also need to update `OpenVINO config`.

Reply:
Updated.